src/test/modelmanager_test.cpp

```diff
@@ -296,13 +297,16 @@ TEST(ModelManager, configReloadNeededChange) {
     modelMock.reset();
 }
 
-TEST(ModelManager, loadConfigManyThreads) {
+TEST_F(ModelManager, loadConfigManyThreads) {
```

Comment:
You use TestWithTempDir fixture, but you are not using test directories so it will have no effect. You have to use `directoryPath' defined in fixture and create files/directories there to have it cleaned up.